src/components/FilterPanel/FilterPanelAccordion/__tests__/FilterPanelAccordion.spec.js

```diff
@@ -9,19 +9,29 @@ import { render } from '@testing-library/react';
 import FilterPanelAccordion from '../FilterPanelAccordion';
 
 describe('FilterPanelAccordion', () => {
-  it('renders with a title', () => {
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
+    render(<FilterPanelAccordion title="test accordion title" />, {
+      // DAP requires a landmark '<main>' in the DOM:
+      container: document.body.appendChild(main),
+    });
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations('FilterPanelAccordion');
```

Comment:
Do all of Axe / DAP tests have the same `expect` statements? What if this whole test function is converted into a utility that reduces the code duplication? Something like:

```jsx
testA11y(<FilterPanelAccordion title="test accordion title" />);
```

It may not be the best idea if the test logic is different in all a11y tests, but I thought I'd make the suggestion.

Reply:
Yeah they all use expect statements. And the only significant detail/difference is that the DAP tests must always have their own unique id (so there can't be multiple `FilterPanelAccordion` tests -- this comes up because some complex components have multiple a11y test blocks. You'll see `Component` and `Component with variation` as ids)

But I imagine even with DAP's unique ID requirement, a utility could still be used. 🤔